.github/CONTRIBUTOR_GUIDE/ct/AppName.md

```diff
@@ -0,0 +1,231 @@
+# **AppName<span></span>.sh Scripts**
+ `AppName.sh` scripts found in the `/ct` directory. These scripts are responsible for the installation of the desired application. For this guide we take `/ct/snipeit.sh` as example.
+
+
+## 1. **File Header**
+
+### 1.1 **Shebang**
+- Use `#!/usr/bin/env bash` as the shebang.
+
+```bash
+#!/usr/bin/env bash
+```
+### 1.2 **Import Functions**
+- Import the build.func file.
+- When developing your own script, change the URL to your own repository.
+
+> [!CAUTION]
+> Before opening a Pull Request, change the URL to point to the community-scripts repo.
+
+Example for development:
+```bash
+source <(curl -s https://raw.githubusercontent.com/[USER]/[REPO]/refs/heads/[BRANCH]/misc/build.func)
+```
+
+Example for final script: 
+```bash
+source <(curl -s https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/misc/build.func)
+```
+
+### 1.3 **Metadata**
+- Add clear comments for script metadata, including author, copyright, and license information.
+
+Example:
+```bash
+# Copyright (c) 2021-2024 community-scripts ORG
+# Author: [YourUserName]
```

Comment:
Why is the author needed?
Afaik by contributing upstream + the git history, this is an outdated information very soon.
I also ask myself in which situation this will help?